app/_includes/braga/00_main.html

```diff
@@ -0,0 +1,20 @@
+<div class="u-paddingLarge"></div>
+
+<section class="CityGuide">
+  <h2 class="SectionTitle">Braga City Guide</h2>
+
+  <div class="CityGuide-intro">
+    <div class="CityGuide-row">
+        <div class="CityGuide-description CityGuide-text">
```

Comment:
Having two descendants in the same element is a smell. We actually don't allow it ;)